observability-lib/grafana/notification-template.go

```diff
@@ -3,21 +3,28 @@ package grafana
 import (
 	"fmt"
 	"os"
+	"path/filepath"
+	"strings"
 
 	"github.com/grafana/grafana-foundation-sdk/go/alerting"
 	"gopkg.in/yaml.v3"
 )
 
-func NewNotificationTemplatesFromFile(filepath string) (map[string]alerting.NotificationTemplate, error) {
+func NewNotificationTemplatesFromFile(filePath string) (map[string]alerting.NotificationTemplate, error) {
+	fileName := strings.TrimSuffix(filepath.Base(filePath), filepath.Ext(filePath))
+	if fileName == "notification-templates" {
+		fileName = "chainlink"
+	}
```

Comment:
why is this necessary ?

Reply:
Because I'm using the filename as the template name. So a `dev-platform-por.yaml` will end up becoming `dev-platform-por-slack-notification-template`.

We can rename `notification-templates.yaml` to `chainlink.yaml` as well. Either way is fine.

Reply:
yes remove that logic I think better to rename file